lib/katello/permission_creator.rb

```diff
@@ -150,50 +149,34 @@ def content_view_permissions # rubocop:disable Metrics/MethodLength
                          :resource_type => 'Katello::ContentView'
     end
 
-    def gpg_key_permissions
-      @plugin.permission :view_gpg_keys,
-                         {
-                           'katello/api/v2/gpg_keys' => [:index, :show, :content, :auto_complete_search]
-                         },
-                         :resource_type => 'Katello::GpgKey'
-      @plugin.permission :create_gpg_keys,
-                         {
-                           'katello/api/v2/gpg_keys' => [:create]
-                         },
-                         :resource_type => 'Katello::GpgKey'
-      @plugin.permission :edit_gpg_keys,
-                         {
-                           'katello/api/v2/gpg_keys' => [:update, :set_content]
-                         },
-                         :resource_type => 'Katello::GpgKey'
-      @plugin.permission :destroy_gpg_keys,
-                         {
-                           'katello/api/v2/gpg_keys' => [:destroy]
-                         },
-                         :resource_type => 'Katello::GpgKey'
-    end
-
     def content_credential_permissions
```

Comment:
`view_gpg_keys` is referenced in lib/katello/plugin.rb and should be `view_content_credentials`

You might also consider updating the `humanized_name` of the GPG Key model to `Content Credentials` so that the Filters UI makes more send (resource called GPG Keys but perm is *_content_credentials...)

Would you also be interested in deprecating the gpg_keys routes in favor of content_credentials? Ya opened up a can of worms here ;)

Reply:
oh man, i thought these were already deprecated. let me do that

Reply:
removed the view_gpg_keys, references, changed humanized_name, deprecated the API's via apipie  properly (previously there was just a log message), and added a migration. 

The migration deletes permissions and moves any existing *gpg_key permissions to content credentials permissions.  The downside is that you may end up with duplicate perms on a filter, but i think thats better than missing them outright.